### PR TITLE
engine-snmp3

### PR DESCRIPTION
**Pull request by noriravi**
SNMPV3 trap configuration for https://bugzilla.redhat.com/show_bug.cgi?id=1268133

### CODE REVIEW

**Comment by mwperina** [comment]
Hmm, looking travis build failure, shouldn't "engine-snmp3" file have '.html.md' suffix?

Other than that it seems to me like good initial page for the feature, we can improve it later.
@oourfali , what do you think?

**Comment by oourfali** [comment]
Looks good to me.
Will send you an email offline for next steps.

File: source/develop/release-management/features/engine/engine-snmp3.html.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# engine-snmp
```
Comment (sandrobonazzola) [inline source/develop/release-management/features/engine/engine-snmp3.html.md]: Missing header with metadata. example:
---
title: CommandCoordinator
category: feature
authors: moti, rnori
feature_name: Introduce CommandCoordinator Framework and Ability To Persist Commands
  in Database
feature_modules: engine
feature_status: Completed
---

File: source/develop/release-management/features/engine/engine-snmp3.html.md [hunk 2]
```diff
@@ -8,6 +16,19 @@ See also [Features/configuration-event-subscribers](Features/configuration-event
...
 This feature extends events notifier capabilities and enables it to generate SNMPv3 traps out of system events to integrate oVirt with generic monitoring systems.
 
+### Owner
+
+*   Name: [Ravi Nori](User:rnori)
+*   Email: <rnori@redhat.com>
+
+### Current status
+
+*   Target Release: 4.0
```
Comment (mwperina) [inline source/develop/release-management/features/engine/engine-snmp3.html.md]: Please change to 4.1

File: source/develop/release-management/features/engine/engine-snmp3.html.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Add a new configuration file /etc/ovirt-engine/notifier/notifier.conf.d/20-snmp.conf to configure the SNMP.
+
+Configuration for sending SNMPv3 NoAuthNoPriv traps as user 'NoAuthNoPriv'.
+
+      SNMP_MANAGERS=localhost
+      SNMP_OID=1.3.6.1.4.1.2312.13.1.1
+      FILTER="include:*(snmp:)"
+      SNMP_VERSION=3
+      SNMP_ENGINE_ID="80:00:00:00:01:02:06:06"
```
Comment (mwperina) [inline source/develop/release-management/features/engine/engine-snmp3.html.md]: Why do we have different id this example than in other two?

File: source/develop/release-management/features/engine/engine-snmp3.html.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+### Current status
+
+*   Target Release: 4.1
+*   Status: Post
+*   Last updated: 
+
+### Configuration
+
+Add a new configuration file /etc/ovirt-engine/notifier/notifier.conf.d/20-snmp.conf to configure the SNMP.
```
Comment (mwperina) [inline source/develop/release-management/features/engine/engine-snmp3.html.md]: Please change file name 99-snmp.conf, AFAIR all user customizations should begin with number higher than 60

File: source/develop/release-management/features/engine/engine-snmp3.html.md (hunk 3 above)
Comment (noriravi) [inline source/develop/release-management/features/engine/engine-snmp3.html.md]: It is a different user. Use NoAuthNoPriv has been configured on Net-snmp to send noAuthNoPriv traps.

File: source/develop/release-management/features/engine/engine-snmp3.html.md (hunk 4 above)
Comment (noriravi) [inline source/develop/release-management/features/engine/engine-snmp3.html.md]: Done

**Comment by mwperina** [comment]
Looks good to me, check failure is unrelated to this pull request. @sandrobonazzola could you please merge?

**Pull request merged**